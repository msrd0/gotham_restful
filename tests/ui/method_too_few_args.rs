#[macro_use] extern crate gotham_restful;

#[derive(Resource)]
#[resource(read)]
struct FooResource;

#[read(FooResource)]
fn read()
{
}

fn main()
{
}
