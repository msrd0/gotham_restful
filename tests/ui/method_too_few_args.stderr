error: Too few arguments
 --> $DIR/method_too_few_args.rs:8:4
  |
8 | fn read()
  |    ^^^^

error[E0433]: failed to resolve: use of undeclared crate or module `_gotham_restful_resource_foo_resource_method_read`
 --> $DIR/method_too_few_args.rs:3:10
  |
3 | #[derive(Resource)]
  |          ^^^^^^^^ use of undeclared crate or module `_gotham_restful_resource_foo_resource_method_read`
  |
  = note: this error originates in a derive macro (in Nightly builds, run with -Z macro-backtrace for more info)
